examples/bls12-381-hash/src/bls12-381-hash.rs

```diff
@@ -0,0 +1,1557 @@
+/* Hashing to Elliptic Curves: https://www.ietf.org/archive/id/draft-irtf-cfrg-hash-to-curve-13.html
+For the BLS12-381 curve.
+Includes both the Shallue-van de Woestijne method and the Simplified Shallue-van de Woestijne-Ulas for AB == 0 method
+for mapping to curve. */
```

Comment:
```suggestion
//! Hashing to Elliptic Curves: <https://www.ietf.org/archive/id/draft-irtf-cfrg-hash-to-curve-13.html>
//! For the BLS12-381 curve.
//! Includes both the Shallue-van de Woestijne method and the Simplified Shallue-van de Woestijne-Ulas for `AB == 0` method
//! for mapping to curve.
```

Reply:
While these comments work in Rust. They are not commonly used. Up here doc comments would be the best.
Can you change the comment style in the other files as well? https://doc.rust-lang.org/rust-by-example/hello/comment.html